addons/account/models/account.py

```diff
@@ -1466,7 +1466,7 @@ def recompute_base(base_amount, fixed_amount, percent_amount, division_amount, p
             tax_amount = round(tax_amount, prec)
             factorized_tax_amount = round(tax_amount * sum_repartition_factor, prec)
 
-            if tax.price_include and not total_included_checkpoints.get(i):
+            if self._context.get('force_price_include') or tax.price_include and not total_included_checkpoints.get(i):
```

Comment:
`if price_include and not total_included_checkpoints.get(i):` ?

Reply:
`price_include` is removed in v13.0, do I still put it here and change it in forward port?
Also context value of `handle_price_include` is True by default but not always, and so, not sure `price_include` will be set correctly every time... what do you think?

Reply:
> `if price_include and not total_included_checkpoints.get(i):` ?

Seems logical

Reply:
Done :+1: